cassandane/cassandane.ini.example

```diff
@@ -95,6 +95,11 @@
 # to run Cassandane using only configuration from environment
 # variables (or defaults)
 ##allow_noinifile = no
+# Perl regular expression to match core file names on your system.
+# The default should match typical default setups, but if you have
+# configured something unusual in sysctl kernel.core_pattern, you
+# should configure this to a regex that will match it.
+##core_pattern = ^core.*?(?:\.\d+)?$
```

Comment:
Same complaint here

Reply:
This is just documentation, so it'll be whatever the actual pattern was above